### ISSUE TITLE
802.11ac @ 80MHz AP mode

### ISSUE THREAD

**Issue by seurat-atreides** [usableprivacy/upribox#138]
Hi,
Is it possible to configure hostapd to operate 802.11ac @ 80MHz?
If so what parameters must I include in the hostapd.conf file?

**Comment by markushuber** [comment]
Hi @seurat-atreides 
We did not implement an option yet to switch the WiFi to AC in 80MHz mode. Would be glad if you report working settings in hostapd.conf back to us, so we can implement an option to switch to AC mode (local changes to hostapd will eventually get overridden by Software updates).

**Comment by tobiasdam** [comment]
802.11ac is only supported by the new Raspberry Pi 3 B+ and there seems to be a [bug](https://github.com/raspberrypi/linux/issues/2619) preventing AC for every country_code other than `country_code=US`

**Comment by seurat-atreides** [comment]
Hi Tobias,
Not true, I'm currently running 802.11ac on an RPI 3 B+ with 40MHz bandwidth for country_code=DE.

**Comment by tobiasdam** [comment]
Since you explicitly stated "80MHz" in the title, I did not mention it in my comment above. The issue also states "5Ghz" (80MHz channel width) in the title and it belongs to an official raspberrypi repository. Even if it would be working for you, it is certainly not working for some others.

**Comment by seurat-atreides** [comment]
Hi *,
for those still interested in getting their Upribox to work as an AP with 80MHz BW.
I did some more testing and got the WLAN IF to work in AP mode with 80MHz BW.
Here is the hostapd.conf file I used:
[hostapd_5G.conf.txt](https://github.com/usableprivacy/upribox/files/2868925/hostapd_5G.conf.txt)

Here is a screenshot from my smartphone:
![upribox_wlan-5g](https://user-images.githubusercontent.com/30745827/52852849-08ea6300-311a-11e9-9b1c-8a4b8a950ca2.jpg)
I had to set the country_code=US instead of DE to get this to work! 
BTW: I removed the metal case cover to get better RF power.
I will be buying a new plastic case.
